app/presenters/dashboard_presenter.rb

```diff
@@ -1,13 +1,30 @@
 class DashboardPresenter
-  def initialize(args)
+  def initialize(args) # rubocop:disable Metrics/AbcSize
     args[:params] = defaults.merge(args[:params].to_hash.symbolize_keys)
     @params = args[:params]
+    return unless args[:params][:query]
```

Comment:
what if query equals {} ? I think that will pass, unless it's expected to pass ?